portfolio/src/main/webapp/script.js

```diff
@@ -12,16 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-window.onload = loadPage;
+//window.onload = loadPage;
```

Comment:
Please consider removing it instead of commenting it out

Reply:
Will do, must have missed this - thanks!